src/twisted/internet/address.py

```diff
@@ -98,7 +98,7 @@ class HostnameAddress(object):
 
 
 
-@attr.s(hash=False, repr=False, cmp=False)
+@attr.s(hash=False, repr=False, eq=False)
```

Comment:
I was wondering what `order` was set to. I see the default is "same as eq", so it is doing the right thing, but I wonder if it's worthwhile calling it out explicitly.

Reply:
The attrs defaults are perfectly sensible now that `cmp` has been split, so I don't think that's necessary.